### ISSUE TITLE
Future of this project?

### ISSUE THREAD

**Issue by davidpetric** [Azure/azure-functions-openapi-extension#645]
Hi,

Is this project deprecated/abandoned?

**Comment by justinyoo** [comment]
No, it's not deprecated nor abandoned. I'm the only maintainer and I have been having other priorities over this.

It's slowly moving forward, though. Thanks for your patience and supports.

**Comment by RuneAndersen** [comment]
@justinyoo What are the expectations regarding version 2.x release, do you have an ETA ?
Also it seems worrying that this is a Microsoft owned project with only one maintainer and you are getting other priorities!

**Comment by vgool-an** [comment]
I'm also eagerly awaiting new updates for this project.

Is there no (rough) timespan you can give for when you expect new versions to be released?

**Comment by Skovvart** [comment]
@justinyoo it would be great with a status update.
If the project is dead, or not looking to pick up pace in a reasonable time frame, the community needs this information to consider alternatives

**Comment by JensSchadron** [comment]
Hii @justinyoo,

First of all thank you for this wonderful library. ❤️

I was wondering however, since the last release was just over 1,5 years ago, whether there's any chance that this project will start merging/reviewing community-contributed PRs and/or addressing pending issues? We're currently in the process of replacing our custom swagger-based solution to this one during our migration from in-process to isolated, but we're stumbling upon issues (like types currently not being allowed to inherit from `List<T>`) for which the community has already opened PRs.

Given that you are seemingly also the only maintainer on this repo, it might be worthwhile to consider adding one (or a few) more maintainers.

Looking forward to a positive response.

**Comment by davidpetric** [comment]
If anyone is interested, someone has developed another package for this use case(I did not use it)

Has already 2.5 million download on nuget.org,  the project is looking for collaborators.

https://github.com/vitalybibikov/AzureExtensions.Swashbuckle

https://www.nuget.org/packages/AzureExtensions.Swashbuckle/

**Comment by CSharpFiasco** [comment]
Looks like there's plans to have a team work on this. https://github.com/Azure/azure-functions-openapi-extension/issues/671#issuecomment-2423424384
